nested.py

```diff
@@ -1,23 +1,62 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Module docstring: One line description of what your program does.
+Program that checks each line in a text file for properly nested brackets
+
 """
-__author__ = "???"
+__author__ = "Rob Spears (GitHub: Forty9Unbeaten)"
 
 import sys
 
 
 def is_nested(line):
-    """Validate a single input line for correct nesting"""
-    pass
+    bracketRef = {'(': ')', '{': '}', '[': ']', '<': '>', '(*': '*)'}
+    openBrackets = bracketRef.keys()
+    closingBrackets = bracketRef.values()
+    balanceStack = []
+    modLine = line
+
+    while modLine:
+        # token determination that accounts for two character tokens
+        # and avoids out of bounds exceptions
+        if len(modLine[0:-1]) > 1:
```

Comment:
I somewhat see what this is doing? I know it works, however:
`if line starts with '(*':` 
`token = '*('`, otherwise `token = line[0]`

Then you can do the same thing you're already doing below, check if it is an open or close bracket, compare, return.